### ISSUE TITLE
try to adjust Gal_Prob

### ISSUE THREAD

**Issue by jradavenport** [RileyWClarke/WideBinaryProject#1]
@kweis suggested we reexamine our cut for Gal_Prob being made, that we might be selecting the *worst* pairs instead of the best.

**Comment by kweis** [comment]
The galactic model from Saurav Dhital outputs probabilities that a pair is a chance alignment. So higher probs = less likely. I noticed on the poster binaries were selected with galprob > 0.001 when really it should be those with galprob < 0.001 (or whatever cutoff). Could have just been a typo (?)

**Comment by jradavenport** [comment]
I downloaded the notebook, tried making this change (make cut Gal_Prob < 0.01). Doesn't look like it greatly changes the result. But that's OK - SCIENCE!

![ab_new](https://cloud.githubusercontent.com/assets/3179868/20114243/5921a3b0-a5a8-11e6-8c68-91578f8d8cc0.png)

However still having troubles when re-runing things, as we saw before NWAM, Riley. I think we've got a bug in the notebook which makes things not reproducible between cells, so we should make a new notebook as described in #2
